.github/workflows/ci.yml

```diff
@@ -183,6 +183,36 @@ jobs:
         env:
           RUSTFLAGS: -D warnings
 
+  cargo-test-kms:
+    runs-on: buildjet-4vcpu-ubuntu-2204
+    timeout-minutes: 45
```

Comment:
One more thing - need to add permissions here:

```
    permissions:
      contents: read
      id-token: write
```

Reply:
Yep, added in a174a8604. The CI workflow is still somewhat rough on the edges, but I'll have to iterate on that here.